nav2_core/include/nav2_core/navigator_exceptions.hpp

```diff
@@ -0,0 +1,47 @@
+// Copyright (c) 2022 Joshua Wallace
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#ifndef NAV2_CORE__NAVIGATOR_EXCEPTIONS_HPP_
+#define NAV2_CORE__NAVIGATOR_EXCEPTIONS_HPP_
+
+#include <stdexcept>
+#include <string>
+
+namespace nav2_core
+{
+
+class NavigatorException : public std::runtime_error
```

Comment:
I think this can all be removed then?

Reply:
As well with the navigate through poses / to pose action msg changes?

Reply:
Exceptions removed.  Keeping some of the constants in the navigate through poses / to pose action msg changes